components/Navigation.vue

```diff
@@ -44,7 +44,7 @@ export default {
           title: 'CPM',
           url: '/cpm',
           children: [{ title: 'Explore', url: '/cpm/explore' },
-            { title: 'Past performance', url: '/cpm/performance' },
+            { title: 'Performance', url: '/cpm/performance' },
```

Comment:
I think the page with past performance/future change should be called "explore". The page that's now called 
"explore" can be changed to "Lines of evidence" or so.

Reply:
The tabs are renamed accordingly 😃.